### PR TITLE
Fix useLocation error

### PR DESCRIPTION
**Pull request by eyalfoni**
## Change description

This PR fixes the error `useLocation() may be used only in the context of a <Router> component.`

## Type of change

- [x] Bug fix (fixes an issue)
- [ ] New feature (adds functionality)
- [ ] Improvement (improves codebase without changing functionality)

## Related issues

Screen Shot 2023-02-13 at 4 38 32 PM

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cortexapps/backstage-plugin",
-  "version": "1.5.0",
+  "version": "1.6.0",
```
Comment (FoodProduct) [inline package.json:3]: Since this is purely a bug fix, we should do a patch version bump for this.
```suggestion
  "version": "1.5.1",
```

**Review by FoodProduct — APPROVED** [review]
While this probably technically fixes the issue (I didnt't test it but I trust you), I think the reason we're in this scenario is the typing is misversioned.

We use `react-router` and `react-router-dom` versions `6.0-beta` but `@types/react-router` and `@types/react-router-dom` versions `5.x` (and `@types/react-router` seems to accidentally be defined as a dependency and not a dev dependency). Unfortunately, it seems like there are no 6.x versions of the type libraries, which makes me wonder if they were rewritten in TypeScript to not need to use definitively typed for a separate package.

tl;dr I'm worried we might regress in the future because there's nothing stopping it, but I'll approve for a quick fix for now
